src/App.js

```diff
@@ -1,16 +1,38 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog'
 
 const App = () => {
+  const [chatEntries, setChatEntries] = React.useState(chatMessages);
+  const setLikes = (id) => {
+    setChatEntries(prevChatEntries => {
+      const updatedChatEntries = prevChatEntries.map(ChatEntry =>{
+        return ChatEntry.id === id ? {...ChatEntry, liked: !ChatEntry.liked} : ChatEntry;
+      })
+      return updatedChatEntries;
+    })
+  }
+  const totalLikes = () =>{
+    let counter = 0
+    for (let chat of chatEntries){
+      if(chat.liked){
+        counter += 1
+      }
+    }
+    return counter
+  };
```

Comment:
This is an optimal approach to calculating the number of likes, great job 😁

This is a great place to display the total likes